### ISSUE TITLE
Cannot focus the accessible buffer after the first time

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#178253]
Using `shift+tab` here, I cannot enter the accessible buffer

https://user-images.githubusercontent.com/29464607/227600462-196c665a-7b48-4d69-a76e-707842a475c3.mov

**Comment by meganrogge** [comment]
ok this happens when `hide` is not applied to `xterm`. 

See here that the buffer editor does get focused here, but you cannot see it


https://user-images.githubusercontent.com/29464607/227614996-c7962d7c-866f-4bb1-bea3-c4d705ed38f8.mov

**Comment by meganrogge** [comment]
Repro:

- run some commands
- `Cmd/Ctrl+shift+o`
- `Escape`
- `Shift+tab`
- `Cmd/Ctrl+shift+o`
- `Escape`
- `Tab`
- Try to `Shift+tab` 🐛 `hide` wasn't applied to `xterm` so the accessible buffer isn't visible

**Comment by meganrogge** [comment]
https://user-images.githubusercontent.com/29464607/227617373-7bbc7a6b-4df1-4e51-a7ba-76fcc0e8a35c.mov
